flake.nix

```diff
@@ -130,6 +132,7 @@
             name = "nix-install-shell";
 
             RUST_SRC_PATH = "${toolchain}/lib/rustlib/src/rust/library";
+            NIX_INSTALLER_TARBALL = inputs.nix.tarballs_direct.${system};
```

Comment:
Is this the raw bytes of the tarball, or just a file path pointing to it (in the Nix store)? If the latter, let's tack on an `_PATH` or something, so that it's clearer.